### ISSUE TITLE
Remove the temp limits (idle/load, min/max) on curves

### ISSUE THREAD

**Issue by marchershey** [Rem0o/FanControl.Releases#2612]
**TLDR: Please uncap the temp limits (idle/load, min/max) on curves.**

Here's my situation...

I want to use my radiator (front) fans, bottom/top fans, and exhaust fans as a "case cooler." 

I wanted to create a custom sensor that gets the sum of all sensors. That comes out to be ~400C at idle, ~520 under load. I wanted to create a trigger curve thats set to 400 idle temp, 500 load temp, (arbitrary, but close) then I can take a graph curve and set markers based off of the sum of all temps, which is much higher than the current limit of 200. 

I understand that I could use averages, but if my chipset or drives are running hotter than normal, and my GPU and CPU are running colder than normal, the averages would be similar to idle temps, but I would want to increase fan speed due to my drives being on fire. 

I hope that makes sense.
